workers/migration.js

```diff
@@ -0,0 +1,52 @@
+//
+// Worker Migration
+//
+
+const logger = require('../libs/bunyan').setModuleName('Migration');
+const rebootCommand = require('../websocket/daemon/reboot');
+const getMacAdressCommand = require('../websocket/daemon/getMacAdress');
+
+function rebootDaemon(userName) {
+  return new Promise((fullfill, reject) => {
+    rebootCommand.exec(userName, () => {
+      logger.info('Reboot started for ' + userName);
+      fullfill()
+    });
+  });
+}
+
+function getMacAdress(userName) {
+  return new Promise((fullfill, reject) => {
+    getMacAdressCommand.exec(userName, (data) => {
+      logger.info('Mac adress for ' + userName);
+      fullfill(data.macAddress)
+    });
+  });
+}
+
+function execCommandDrbl(imagePath, macAddress) {
```

Comment:
As this function is not yet implemented, I think that you should `reject('not implemented');`